app/main.py

```diff
@@ -1 +1,29 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command_list = command.split(" ")
```

Comment:
`" "` is the default separator for .split(). Use split() without arguments instead